checkov/kustomize/runner.py

```diff
@@ -229,7 +229,7 @@ def check_system_deps(self):
                         self.templateRendererCommand = self.kubectl_command
                         return None
             
-            except Exception:
+            except Exception as e:
                 logging.debug(f"An error occured testing the {self.kubectl_command} command: {e}")
```

Comment:
```suggestion
            except Exception:
                logging.debug(f"An error occured testing the {self.kubectl_command} command", exc_info=True)
```
then we have the complete stack trace